apps/src/sites/studio/pages/blocks/index.js

```diff
@@ -0,0 +1,52 @@
+import $ from 'jquery';
+import assetUrl from '@cdo/apps/code-studio/assetUrl';
+import jsonic from 'jsonic';
+import { parseElement } from '@cdo/apps/xml';
+import { installCustomBlocks } from '@cdo/apps/block_utils';
+import { customInputTypes } from '@cdo/apps/gamelab/blocks';
+import { valueTypeTabShapeMap } from '@cdo/apps/gamelab/GameLab';
+import { shrinkBlockSpaceContainer } from '@cdo/apps/templates/instructions/utils';
+import animationListModule, {
+  setInitialAnimationList
+} from '@cdo/apps/gamelab/animationListModule';
+import defaultSprites from '@cdo/apps/gamelab/defaultSprites.json';
```

Comment:
you're importing a lot of stuff from gamelab for this; is it intentional that this preview is gamelab-specific?

Reply:
Those imports are required to render the block previews correctly for the GameLabJr pool. There might be a way to dynamically import them only if the pool is GameLabJr, but I'm not sure that it's necessary in this case.

Reply:
I'm more concerned about the fact that we're _not_ loading this for anything that's not gamelab; notably, Dancelab also has things like a `valueTypeTabShapeMap` that aren't getting included here

Reply:
it's arguably out of scope for this PR to make this broadly compatible, but is at the very least worth tracking as a long-term work item somewhere

Reply:
Oh I see. At least for right now, Dancelab's valueTypeTabShapeMap is a subset of the one from Gamelab. But I agree that it could potentially be a problem. Is there a place to track things like this?

Reply:
I'm not sure where the best place is to track stuff while we're in this transitional sprint. @ryansloan might have an answer, though

Reply:
I will add to the backlog to ensure it gets imported into Jira. This is off to an _awesome_ start!!